README.md

```diff
@@ -45,12 +44,11 @@ apiInstance.getAccount().then(function(data) {
 ```javascript
 const SibApiV3Sdk = require('sib-api-v3-typescript');
 
-let apiInstance = new SibApiV3Sdk.ContactsApi()
+const apiInstance = new SibApiV3Sdk.ContactsApi()
 
 // Configure API key authorization: apiKey
 
-let apiKey = apiInstance.authentications['apiKey'];
-apiKey.apiKey = "YOUR API KEY"
+apiInstance.setApiKey(SibApiV3Sdk.AccountApiApiKeys.apiKey, 'YOUR API KEY');
```

Comment:
`AccountApiApiKeys` generates un error

it should be `ContactsApiApiKeys`

Reply:
Indeed! Updated 😉